Userland/Libraries/LibWeb/CSS/StyleValues/RotationStyleValue.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2024, Steffen T. Larssen <dudedbz@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGfx/Vector3.h>
+#include <LibWeb/CSS/Angle.h>
+#include <LibWeb/CSS/StyleValues/CSSUnitValue.h>
+
+namespace Web::CSS {
+
+class RotationStyleValue : public CSSUnitValue {
```

Comment:
I don't think it makes sense for this to be a CSSUnitValue. You can't do arithmetic on it.